Add DCO and contributor guide link (#36)
Fitbit/fitbit-sdk-toolchain: CONTRIBUTING.md

> Visit [dev.fitbit.com](https://dev.fitbit.com/community/contributing/) to find the current version of our contributor guide.

Reviewer: It doesn't feel right to have a deep-link to the guide with a partial URL as the text. How about this copy? Bonus: it is just as readable in plain text as in rendered Markdown.
```suggestion
The current version of our contributing guide can be found at https://dev.fitbit.com/community/contributing/.
```
